loops_lab.js

```diff
@@ -0,0 +1,239 @@
+// // # Loops Lab
+// //
+// // To complete this lab, do the following:
+// //
+// // 1. Fork this repo
+// // 2. Clone **your** forked repo
+// // 3. Create a file called `loops_lab.js` inside your cloned repo
+// // 4. Copy the description of the exercise as a comment
+// // 5. Underneath the commented description write your code.
+// // 6. Ensure your solutions work properly!
+// // 7. Open PR against this repo.
+// //
+// // ___
+// //
+// // ## Exercises
+// //
+// // 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+//
+// //descending variable with while loop
+let num = 44;
+while (num >=1){
+  console.log(num);
+  num--;
+}
+
+// // descending variable with for loop (used different var for same numbers)
+for(let x = 44; x >= 1; x --){
+  console.log(x);
+}
+
+
+// // 2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+// // For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+// // while loop with iteration
+let o = 11;
+while(o >= 1 ){
+  if(o % 2 === 0){
+    console.log(o + ' is even.');
+  } else {
+    console.log(o + ' is odd.');
+  }
+  o--;
+}
+
+// //for loop with iteration
+for(let i = 11; i >= 1; i--){
+  if(i % 2 === 0){
+    console.log(i + ' is even.')
+  } else {
+    console.log(i + ' is odd.')
+  }
+}
+
+// // // // 3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
+// // // // For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
+// //
+// // while iteration
+let num = 1;
+let it = 9;
```

Comment:
A very important good practice is to have good variable name. "it" and "sol" were confusing for me to read and made me wonder if thoese were abbreviation for something.